sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -316,7 +318,7 @@ describe("Atom management - Authentication", function(): void {
       );
       should.equal(
         (await serviceBusAdministrationClient.getNamespaceProperties()).name,
-        host.match("(.*).servicebus.windows.net")[1],
+        (host.match("(.*).servicebus.windows.net") || [])[1],
```

Comment:
Is this going to work when we get to the "private" clouds?

Reply:
No, it wont. But since our tests do not run in private clouds, we should be good. We should make sure we dont do this in any of our library code though